docs/content/docs/build/presence.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Presence
+aliases:
+  - "/docs/build/presence/"
+author: pranshu
+---
+
+## Overview
+
+We are introducting a new way to power your ephemral experiences wth Fluid. Introducing the new Presence APIs (currently in alpha) that provide session-focused utilities for lightweight data sharing and messaging.
+
+Collaborative features typically rely on each user maintaining their own temporary state, which is subsequently shared with others. For example, in applications featuring multiplayer cursors, the cursor position of each user signifies their state. This state can be further utilized for various purposes such as indicating typing activity or displaying a user's current selection. This concept is referred to as Presence.
+
+By leveraging this shared state, applications can provide a seamless and interactive collaborative experience, ensuring that users are always aware of each other's actions and selections in real-time.
+
+The key scenarios that the new Presence APIs are suitable for includes:
+- User presence
+- Collaborative cursors
+- Notifications
+
+
+## Concepts
+
+A session is a period of time when one or more clients are connected to a Fluid service. Session data and messages may be exchanged among clients, but will disappear once the no clients remain. (More specifically once no clients remain that have acquired the session `IPresence` interface.) Once fully implemented, no client will require container write permissions to use Presence features.
```

Comment:
As I understand it, each client has its own session with the service. But it looks like you are talking about some kind of shared session that multiple clients are part of. If that's the case, let's make sure this new sense of "session" is distinct. I recommend we use the term "presence session" or even "shared presence session" or "common presence session" whenever we're talking about session in this new sense.

Reply:
Will change in the next iteration